libs/liblobby/lobby/lobby.lua

```diff
@@ -652,7 +652,7 @@ function Lobby:_OnBattleOpened(battleID, type, natType, founder, ip, port,
 	self.battles[battleID] = {
 		battleID = battleID, type = type, natType = natType, founder = founder, ip = ip, port = port,
 		maxPlayers = maxPlayers, passworded = passworded, rank = rank, mapHash = mapHash, spectatorCount = spectatorCount or 0,
-		engineName = engineName, engineVersion = engineVersion, mapName = mapName, title = title, gameName = gameName, users = {},
+		engineName = engineName, engineVersion = engineVersion, mapName = mapName, title = title, gameName = gameName, users = {founder},
```

Comment:
the whole thing is a bit messy, i'll restructure it a bit and synchronize when i'm done